db/schema.rb

```diff
@@ -56,6 +56,19 @@
     t.text "description"
     t.datetime "created_at", null: false
     t.datetime "updated_at", null: false
+    t.string "onet_code"
+    t.string "rapids_code"
+    t.integer "time_based_hours"
+    t.string "hybrid_hours"
+    t.integer "competency_based_hours"
+  end
```

Comment:
@jenlu33 can you remove this from the PR since it is actually part of the other PR with the rapids data import?

Reply:
@littleforest Is there a way for me to get rid of the migration in the branch? Or do I just manually remove those lines? Since it always reappears when I run `rake db:migrate`